mmd_tools/core/model.py

```diff
@@ -1540,7 +1485,7 @@ def __createNonCollisionConstraint(self, nonCollisionJointTable):
         logging.debug("-" * 60)
         logging.debug(" creating ncc, counts: %d", total_len)
 
-        ncc_obj = bpyutils.createObject(name="ncc", object_data=None)
+        ncc_obj = createObject(name="ncc", object_data=None)
```

Comment:
I suggest using `FnContext.new_and_link_object` instead of `bpyutils.createObject`